vnext/template/cpp-app/proj/MyApp.sln

```diff
@@ -24,8 +24,6 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Microsoft.ReactNative", "..
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "JSI.Shared", "..\node_modules\react-native-windows\JSI\Shared\JSI.Shared.vcxitems", "{0CC28589-39E4-4288-B162-97B959F8B843}"
```

Comment:
>JSI.Shared [](start = 53, length = 10)

Should it be removed too?

Reply:
The same is for the other new solution templates.

---
In reply to: [547330556](https://github.com/microsoft/react-native-windows/pull/6783#discussion_r547330556) [](ancestors = 547330556)